runtime/src/test/java/com/flipkart/flux/dao/DummyOnEntryHook.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2012-2016, the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.flipkart.flux.dao;
+
+import com.flipkart.flux.domain.Event;
+import com.flipkart.flux.domain.Hook;
+
+/**
+ * @author shyam.akirala
+ */
+public class DummyOnEntryHook implements Hook {
```

Comment:
I wanted to store the proper fqn of class in db, and retrieve back and load the class using reflection. Not sure using mocks I can do this.